rs/nns/sns-wasm/src/sns_wasm.rs

```diff
@@ -1085,56 +1090,53 @@ where
     }
 
     /// Remove the SNS wasm canister as the controller of the canisters
-    async fn remove_self_as_controller(
+    async fn remove_sns_w_as_controller(
```

Comment:
This issue predates this PR, but...

Alternatively, this could be named something like `make_sns_canisters_control_each_other_exclusively`.

This only "removes" if we are the only other controller (before this is called). This is an unstated assumption of this method.

Reply:
Note that in your review, you suggest not including changes that are optional (to highlight the most essential change, which makes sense) and you also suggest a renaming that is nice to have (to make the code clearer, which would be nice to have). So I take the liberty to interpret these suggestions liberally, and keeping this change as-is for now.